cmd/snap-seccomp/main.go

```diff
@@ -841,21 +867,33 @@ func compile(content []byte, out string) error {
 	}
 
 	if osutil.GetenvBool("SNAP_SECCOMP_DEBUG") {
-		secFilter.ExportPFC(os.Stdout)
+		secFilterAllow.ExportPFC(os.Stdout)
+		secFilterDeny.ExportPFC(os.Stdout)
 	}
 
 	// write atomically
```

Comment:
with two files this is no longer atomic so we should probably use a dir here

Reply:
I tweaked this a bit now to make the seccomp profile generation still atomic. Instead of a file it will use a directory that is then renamed via renameat2() in an atomic way.